vxdetector/interact_bowtie2.py

```diff
@@ -1,30 +1,70 @@
 #!/usr/bin/python
 
 import os
-from os.path import exists
+# from Output_counter import directory_navi
 
 
 def buildbowtie2(path):
-    bowtie2_path = '/vol/software/bin/bowtie2-build'
```

Comment:
we will need a better mechanism to specify binary program locations like a config file or cmd line arguments